### PR TITLE
[BULK UPDATES] DocuTune - Fix build validation issues: docs-link-absolute

### PR DESCRIPTION
**Pull request by alexbuckgit**
**Global effort to fix build validation errors**

The Microsoft Skilling team is fixing build validation errors and specific brand inconsistencies in Microsoft technical documentation for the rest of FY23 Q1. This will help address various accessibility, security, and usability issues. This PR was generated using DocuTune. It includes only targeted fixes and minor formatting adjustments.

DocuTune PRs improve quality of technical content by finding and automatically correcting a broad set of issues only where the correction is suitable. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

CorrelationId: 0df9c2d6-63f8-4d1e-8b57-b33611a06514
InitiativeId: docutune-build-validation-issues-2022-05
PointOfContact: Alex Buck

### CODE REVIEW

**Review by 333fred — APPROVED** [review]

**Comment by brittmsantos** [comment]
#docutune-ready

**Comment by alexbuckgit** [comment]
@333fred @adamperlin @AlekseyTs @cston @DaZombieKiller @jaredpar @jcouv @jnm2 @jnyrup @MadsTorgersen @peter-dolkens @xclud @Youssef1313

Please merge the changes in this PR. For more information, see the description at the top of this PR.

Any pre-existing build validation issues or insufficient Acrolinx scores should be addressed in a separate PR.

#docutune-request-approval

**Review by jnm2 — APPROVED** [review]
These all look great to me.

**Review by BillWagner — APPROVED** [review]
Thanks folks. 

I had a couple comments, and then this can be merged.

None of the files in this PR are published, so it's not as critical for the docs validation issues.

File: meetings/2021/LDM-2021-09-15.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ have an initial conversion of an internal Microsoft C# specification of C# 3, wh
 team has converted the C# 5 specification to markdown, and is currently working to integrate the changes from our C# 6 spec into that
 version.
 
-These dual versions of the specification can confuse users. The language reference on docs.microsoft.com comes from the csharplang
+These dual versions of the specification can confuse users. The language reference on Microsoft Docs comes from the csharplang
```
[the remark this answers is not in the document]
Comment (333fred) [reply meetings/2021/LDM-2021-09-15.md:24]: Yeah, I don't think we should make any more changes.

**Comment by alexbuckgit** [comment]
> @alexbuckgit most of the people you tagged in that list do not have merge permissions in this repo. Where did you pull it from?

@333fred -
It's pulled from the last 30 commits in the repo. I'll use that criteria to @ mention enough people so that the PR is seen and merged. Once I've found the right contacts for the repo, I update our contact list.

Thanks for merging!

**Comment by 333fred** [comment]
> It's pulled from the last 30 commits in the repo. I'll use that criteria to @ mention enough people so that the PR is seen and merged. Once I've found the right contacts for the repo, I update our contact list.

Consider making sure that the people you tag are actually members of the repo. It's a bit rude to mass tag people like that, especially since many of those people are our customers who were kind enough to send a pr to fix some issue or another.

**Pull request merged**